LegitimateURLShortener.txt

```diff
@@ -1847,6 +1847,14 @@ $removeparam=iv_,domain=allbirds.com
 ||weidian.com^$removeparam=ifr
 ||weidian.com^$removeparam=share_relation
 
+! https://www.etsy.com/listing/655283263/docking-stationgift-for-men-for?organic_search_click=1&pop=1
+! https://www.etsy.com/listing/984664013/personalized-leather-walletmens?plkey=e61127eaa78c1a3d517e415d3e2014db7e43e6f5%3A984664013&pro=1&frs=1
+$removeparam=organic_search_click,domain=etsy.com
```

Comment:
```suggestion
$removeparam=organic_search_click,domain=etsy.*
```
Probably this should use a wildcard TLD as to work for other countries.

Reply:
I checked, and the addresses for different countries seem to be like that:
```
https://www.etsy.com/au/
https://www.etsy.com/ca/
https://www.etsy.com/no-en/
https://www.etsy.com/se-en/
....
```

Reply:
I did find one on another TLD but it just redirected